_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -94,7 +94,17 @@ _There are no major accessibility support issues known for this rule._
 `aria-owns` property with valid ID reference list value
 
 ```html
-<div role="combobox" aria-owns="my-textbox my-grid" aria-label="Search within the website"></div>
+<h1>Shopping list</h1>
+<div role="list" aria-owns="item1 item2"></div>
+<div id="item1">
+	Apples
+	<div>
+		<div id="item2">
+			Bananas
+			<div></div>
+		</div>
+	</div>
+</div>
```

Comment:
I fail to see the usefulness of so many `div`s… Would
```suggestion
<h1>Shopping list</h1>
<div role="list" aria-owns="item1 item2"></div>
<div id="item1">Apples</div>
<div id="item2">Bananas</div>
```
be enough to illustrate this example?

Reply:
seems reasonable.